### ISSUE TITLE
branch: jump

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/faradays-law#104]
@mbarlow12 mentioned this branch in his 2/8/18 status report.

FYI, all non-release branches are supposed to have an associated GitHub issue. Title is of the form "branch: {{branchName}}}".  The description should include: the purpose of the branch, expected lifetime of the branch, references to any related issues.

**Comment by mbarlow12** [comment]
@pixelzoom Thanks for the info. Is it worth adding this to the [phet-info doc](https://github.com/phetsims/phet-info/blob/f9cae306d2c90c4b15abc122681e12cb2f9dde34/deployment-info/chipper-2.0.md)? If so, I'd be happy to create an issue and update.

**Comment by mbarlow12** [comment]
Keyboard jump is a proposed interaction for Faraday's Law where the user can simply use the 'j' key to allow the magnet to animate to a reflected position across the page. This would allow keyboard navigation users to generate results on the voltmeter in a more controlled fashion. For design and discussion see issue #103.

This branch will likely last a few weeks. Once we solidify the desired behavior, we can merge into master.

**Comment by pixelzoom** [comment]
> Is it worth adding this to the phet-info doc?

Yes, that would be useful.  But there are also non-build situations where people create branches.

> This branch will likely last a few weeks. 

I should also have mentioned... An issue like this should be closed if/when its branch is deleted.

**Comment by mbarlow12** [comment]
Renamed branch to keyboard-jump. Underscores are not allowed in the semantic versioning spec , and npm was throwing an error.

**Comment by mbarlow12** [comment]
Jon has outlined some new interaction spec for the jump/launch behavior in https://wiki.fluidproject.org/display/fluid/PhET+Faraday%27s+Law+Simulation+-+Text+descriptions%2C+alerts%2C+and+keyboard+access.

Updating the behavior to match.

**Comment by mbarlow12** [comment]
Renamed again to `jump` for one-off deployment.

**Comment by jbphet** [comment]
I'm going through the issues in this repo in preparation for an upcoming publication with a11y features.  When I look at the branch described by this issue, the summary says "This branch is 494 commits behind master" and doesn't say anything about having any commits that are ahead of master.  Also, the keyboard jump behavior - or at least some form of it - is currently operational on master.  Based on this evidence, I'm going to conclude that this branch was merged into master, and the branch can now be deleted.

**Comment by jbphet** [comment]
Here's a screenshot of a portion of the branches page:

![image](https://user-images.githubusercontent.com/3082831/97374074-10658780-187d-11eb-9372-dcccbc179184.png)

**Comment by jbphet** [comment]
Branch deleted, closing.

**Comment by jbphet** [comment]
Reopening.  This branch was restored during an effort to recover some branches that were accidentally deleted.  This one had been _intentionally_ deleted, and now needs to be re-deleted.  See https://github.com/phetsims/special-ops/issues/198.

**Comment by jbphet** [comment]
Re-deletion complete.  Closing.
